<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreateReportWidget</class>
 <widget class="QWidget" name="CreateReportWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1289</width>
    <height>1039</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>560</y>
     <width>321</width>
     <height>291</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget {
    background-color: #007BFF; /* Синий фон для окна */
    border-radius: 12px; /* Закругленные углы */
}</string>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>231</width>
      <height>231</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QCheckBox" name="checkSNP_2">
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="mouseTracking">
        <bool>true</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">QCheckBox {
    color: white;
    font-size: 18px;
    font-weight: bold;
    spacing: 8px;
}

QCheckBox::indicator {
    width: 18px;
    height: 18px;
    border-radius: 4px;
    background-color: white; /* Изначально белый фон */
    border: 2px solid #1E90FF; /* Граница для видимости */
}

QCheckBox::indicator:unchecked {
    background-color: white; /* Белый фон, когда не выбран */
}

QCheckBox::indicator:checked {
    background-color: #00FF00; /* Тёмно-синий фон при выборе */
}

QCheckBox::indicator:checked:pressed {
    background-color: #000080; /* Темно-синий фон при нажатии */
}

QCheckBox::indicator:hover {
    background-color: #00BFFF; /* Светло-синий при наведении */
}

/* Убираем лишние стили для галочки */
QCheckBox::indicator:checked {
    border: 2px solid #1E90FF; /* Граница при выборе */
}</string>
       </property>
       <property name="text">
        <string>ФИО</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkAge_2">
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">QCheckBox {
    color: white;
    font-size: 18px;
    font-weight: bold;
    spacing: 8px;
}

QCheckBox::indicator {
    width: 18px;
    height: 18px;
    border-radius: 4px;
    background-color: white; /* Изначально белый фон */
    border: 2px solid #1E90FF; /* Граница для видимости */
}

QCheckBox::indicator:unchecked {
    background-color: white; /* Белый фон, когда не выбран */
}

QCheckBox::indicator:checked {
    background-color: #00FF00; /* Тёмно-синий фон при выборе */
}

QCheckBox::indicator:checked:pressed {
    background-color: #000080; /* Темно-синий фон при нажатии */
}

QCheckBox::indicator:hover {
    background-color: #00BFFF; /* Светло-синий при наведении */
}

/* Убираем лишние стили для галочки */
QCheckBox::indicator:checked {
    border: 2px solid #1E90FF; /* Граница при выборе */
}</string>
       </property>
       <property name="text">
        <string>Возраст</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkPhone_2">
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">QCheckBox {
    color: white;
    font-size: 18px;
    font-weight: bold;
    spacing: 8px;
}

QCheckBox::indicator {
    width: 18px;
    height: 18px;
    border-radius: 4px;
    background-color: white; /* Изначально белый фон */
    border: 2px solid #1E90FF; /* Граница для видимости */
}

QCheckBox::indicator:unchecked {
    background-color: white; /* Белый фон, когда не выбран */
}

QCheckBox::indicator:checked {
    background-color: #00FF00; /* Тёмно-синий фон при выборе */
}

QCheckBox::indicator:checked:pressed {
    background-color: #000080; /* Темно-синий фон при нажатии */
}

QCheckBox::indicator:hover {
    background-color: #00BFFF; /* Светло-синий при наведении */
}

/* Убираем лишние стили для галочки */
QCheckBox::indicator:checked {
    border: 2px solid #1E90FF; /* Граница при выборе */
}</string>
       </property>
       <property name="text">
        <string>Номер телефона</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBlock_2">
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">QCheckBox {
    color: white;
    font-size: 18px;
    font-weight: bold;
    spacing: 8px;
}

QCheckBox::indicator {
    width: 18px;
    height: 18px;
    border-radius: 4px;
    background-color: white; /* Изначально белый фон */
    border: 2px solid #1E90FF; /* Граница для видимости */
}

QCheckBox::indicator:unchecked {
    background-color: white; /* Белый фон, когда не выбран */
}

QCheckBox::indicator:checked {
    background-color: #00FF00; /* Тёмно-синий фон при выборе */
}

QCheckBox::indicator:checked:pressed {
    background-color: #000080; /* Темно-синий фон при нажатии */
}

QCheckBox::indicator:hover {
    background-color: #00BFFF; /* Светло-синий при наведении */
}

/* Убираем лишние стили для галочки */
QCheckBox::indicator:checked {
    border: 2px solid #1E90FF; /* Граница при выборе */
}</string>
       </property>
       <property name="text">
        <string>Номер блока</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkOpt_2">
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">QCheckBox {
    color: white;
    font-size: 18px;
    font-weight: bold;
    spacing: 8px;
}

QCheckBox::indicator {
    width: 18px;
    height: 18px;
    border-radius: 4px;
    background-color: white; /* Изначально белый фон */
    border: 2px solid #1E90FF; /* Граница для видимости */
}

QCheckBox::indicator:unchecked {
    background-color: white; /* Белый фон, когда не выбран */
}

QCheckBox::indicator:checked {
    background-color: #00FF00; /* Тёмно-синий фон при выборе */
}

QCheckBox::indicator:checked:pressed {
    background-color: #000080; /* Темно-синий фон при нажатии */
}

QCheckBox::indicator:hover {
    background-color: #00BFFF; /* Светло-синий при наведении */
}

/* Убираем лишние стили для галочки */
QCheckBox::indicator:checked {
    border: 2px solid #1E90FF; /* Граница при выборе */
}</string>
       </property>
       <property name="text">
        <string>Часы ОПТ</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>10</y>
      <width>241</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #FFFFFF;
font-size: 24px; /* Размер шрифта кнопки */
font-weight: bold; /* Жирный текст */</string>
    </property>
    <property name="text">
     <string>Содержание отчёта</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="createReportButton">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>950</y>
     <width>281</width>
     <height>61</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: #0056b3; /* Основной цвет фона кнопки */
    color: white; /* Цвет текста кнопки */
    border: none; /* Убираем рамки */
    border-radius: 12px; /* Закругленные углы */
    padding: 5px; /* Отступы внутри кнопки */
    font-size: 22px; /* Размер шрифта кнопки */
    font-weight: bold; /* Жирный текст */
    margin-top: 5px; /* Отступ сверху */
}

QPushButton:hover {
    background-color: #004085; /* Темнее при наведении */
}

QPushButton:pressed {
    background-color: #003366; /* Цвет при нажатии */
}</string>
   </property>
   <property name="text">
    <string>Создать отчёт</string>
   </property>
  </widget>
  <widget class="QScrollArea" name="scrollArea">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>40</y>
     <width>931</width>
     <height>901</height>
    </rect>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>929</width>
      <height>899</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="searchButton">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>950</y>
     <width>251</width>
     <height>61</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: #0056b3; /* Основной цвет фона кнопки */
    color: white; /* Цвет текста кнопки */
    border: none; /* Убираем рамки */
    border-radius: 12px; /* Закругленные углы */
    padding: 5px; /* Отступы внутри кнопки */
    font-size: 22px; /* Размер шрифта кнопки */
    font-weight: bold; /* Жирный текст */
    margin-top: 5px; /* Отступ сверху */
}

QPushButton:hover {
    background-color: #004085; /* Темнее при наведении */
}

QPushButton:pressed {
    background-color: #003366; /* Цвет при нажатии */
}</string>
   </property>
   <property name="text">
    <string>Поиск</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>321</width>
     <height>351</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget {
    background-color: #007BFF; /* Синий фон для окна */
    border-radius: 12px; /* Закругленные углы */
}</string>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>10</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #FFFFFF;
font-size: 24px; /* Размер шрифта кнопки */
font-weight: bold; /* Жирный текст */</string>
    </property>
    <property name="text">
     <string>Фильтры</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="patronymEdit_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>200</y>
      <width>301</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
    background-color: #FFFFFF; /* Белый фон для полей ввода */
    border: 2px solid #0056b3; /* Синяя рамка */
    border-radius: 4px; /* Закругленные углы */
    padding: 2px; /* Отступы внутри поля */
    font-size: 18px; /* Размер шрифта */
}

QLineEdit:focus {
    border-color: #007BFF; /* Цвет рамки при фокусе */
}</string>
    </property>
    <property name="placeholderText">
     <string>Отчество</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="nameEdit_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>301</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
    background-color: #FFFFFF; /* Белый фон для полей ввода */
    border: 2px solid #0056b3; /* Синяя рамка */
    border-radius: 4px; /* Закругленные углы */
    padding: 2px; /* Отступы внутри поля */
    font-size: 18px; /* Размер шрифта */
}

QLineEdit:focus {
    border-color: #007BFF; /* Цвет рамки при фокусе */
}</string>
    </property>
    <property name="placeholderText">
     <string>Имя</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="blockEdit_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>280</y>
      <width>301</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
    background-color: #FFFFFF; /* Белый фон для полей ввода */
    border: 2px solid #0056b3; /* Синяя рамка */
    border-radius: 4px; /* Закругленные углы */
    padding: 2px; /* Отступы внутри поля */
    font-size: 18px; /* Размер шрифта */
}

QLineEdit:focus {
    border-color: #007BFF; /* Цвет рамки при фокусе */
}</string>
    </property>
    <property name="placeholderText">
     <string>Номер блока</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="surnameEdit_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>301</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
    background-color: #FFFFFF; /* Белый фон для полей ввода */
    border: 2px solid #0056b3; /* Синяя рамка */
    border-radius: 4px; /* Закругленные углы */
    padding: 2px; /* Отступы внутри поля */
    font-size: 18px; /* Размер шрифта */
}

QLineEdit:focus {
    border-color: #007BFF; /* Цвет рамки при фокусе */
}</string>
    </property>
    <property name="placeholderText">
     <string>Фамилия</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
